### DISCUSSION TITLE
Popover snippet for vue

### DISCUSSION THREAD

**Discussion by Extarys** [chakra-ui/ark#823] [Q&A] [answered]
Can someone share a snippet on how to properly use Popover with Vue? (The docs doesn't have it for this particular component)

I wonder if it would be possible to open the Popover by listening to `mouseover` the trigger.

Thanks community :)

**Comment by Shyrro** [comment]
Hey @Extarys 

Thanks for reaching out.

Try this snippet and let me know : 

```html
<script setup lang="ts"> 
 import { 
   Popover, 
   PopoverArrow, 
   PopoverArrowTip, 
   PopoverCloseTrigger, 
   PopoverContent, 
   PopoverDescription, 
   PopoverPositioner, 
   PopoverTitle, 
   PopoverTrigger, 
 } from '@ark-ui/vue' 
 </script> 
 <template> 
   <Popover> 
     <PopoverTrigger> 
       <button>click me</button> 
     </PopoverTrigger> 
     <PopoverPositioner> 
       <PopoverArrow> 
         <PopoverArrowTip /> 
       </PopoverArrow> 
       <PopoverContent> 
         <PopoverTitle>Title</PopoverTitle> 
         <PopoverDescription>Description</PopoverDescription> 
         <input type="text" /> 
         <PopoverCloseTrigger> 
           <button>close</button> 
         </PopoverCloseTrigger> 
       </PopoverContent> 
     </PopoverPositioner> 
   </Popover> 
 </template>
```

**Reply by Extarys** [reply]
It's working, thank you very much! Any idea if it's possible to open the popover on mouseover? Looked at the code a bit, but I'm unsure how it's suppose to work :joy:

**Reply by Shyrro** [reply]
Hey @Extarys , 

You can attach any events you like to the Popover components. To open/close the popover, you can use the `isOpen` prop. 
You can change that value onmouseover if you want. 

Hope that helps !

**Comment by anubra266** [comment, marked as answer]
@Extarys You can play around with it here
https://arkxamples.vercel.app/popover
